pvactools/lib/generate_reviews_files.py

```diff
@@ -50,7 +52,62 @@ def make_column_unique(df, column_name):
     return df
 
 
-def main(peptides_path, classI_path, classII_path, sample_name, all_epitopes_flag):
+# Function that reads a vcf and returns the variants that contain "PASS" in "FILTER"
+def load_pass_variants(vcf_path):
+    variants = set()
+    reader = vcfpy.Reader.from_path(vcf_path)
+    for record in reader:
+        if record.FILTER and record.FILTER[0] != "PASS":
```

Comment:
I think this logic will also process any variants without a FILTER (please double check me on this). I think that behavior is ok/desired because an empty FILTER is generally considered the same as PASS.

Reply:
Ok, that sounds reasonable to me. The edit I made to the help text for the external VCF also tries to make that clear.